packages/build-tools/src/steps/utils/ios/fastlane.ts

```diff
@@ -0,0 +1,81 @@
+import path from 'path';
```

Comment:
I'm not convinced this is a good dir structure, but I think the best approach is to move stuff around when we have full build flow implemented and leave if for now how it is.